fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/IcebergUtils.java

```diff
@@ -665,4 +668,16 @@ public static String dataLocation(Table table) {
         }
         return dataLocation;
     }
+
+    public static HiveCatalog createIcebergHiveCatalog(ExternalCatalog externalCatalog, String name) {
+        HiveCatalog hiveCatalog = new org.apache.iceberg.hive.HiveCatalog();
+        hiveCatalog.setConf(externalCatalog.getConfiguration());
+
+        Map<String, String> catalogProperties = externalCatalog.getProperties();
+        String metastoreUris = catalogProperties.getOrDefault(HMSProperties.HIVE_METASTORE_URIS, "");
+        catalogProperties.put(CatalogProperties.URI, metastoreUris);
+
+        hiveCatalog.initialize(name, catalogProperties);
+        return hiveCatalog;
```

Comment:
Is same as `IcebergHMSExternalCatalog.initCatalog` ?

Reply:
yes . so now  IcebergHMSExternalCatalog.initCatalog() method use this method.

Reply:
you can ref : https://github.com/apache/doris/pull/41659/files#diff-e6e9c0730301ce99354da7a68c322f1c94d656033b25ab525eab6f17eb9dce7dL41